tests/integration_tests.py

```diff
@@ -96,3 +98,38 @@ def test_callproc_mod_result_value(self):
             raise unittest.SkipTest('PostgreSQL is not running')
         self.assertEqual(6 % 4, result[0]['mod'])
         result.free()
+
+    @testing.gen_test
+    def test_polling_stops_after_connection_error(self):
+        # Abort the test right away if postgres isn't running.
+        good_uri = queries.uri('localhost', 5432, 'postgres', 'postgres')
+        try:
+            test_session = queries.Session(good_uri, pool_max_size=10)
+            test_session.close()
+        except queries.OperationalError as error:
+            raise unittest.SkipTest(str(error).split('\n')[0])
```

Comment:
I think I'd rather assert a failure here. Otherwise is there any harm in continuing to the next part of the test?

Reply:
@gmr No harm in it, but the rest of the tests in this file all skip if postgres isn't running.